polynote-server/src/main/scala/polynote/server/Server.scala

```diff
@@ -94,25 +93,21 @@ class Server(kernelFactory: Kernel.Factory.Service) extends polynote.app.App {
           |
           |""".stripMargin
 
-  override def main(args: List[String]): ZIO[Environment, Nothing, Int] = main.provideSomeLayer[Environment] {
-    Server.parseArgs(args).orDie andThen globalEnv.orDie
-  }
-
-  def main: ZIO[MainArgs with BaseEnv with ServerEnv, Nothing, Int] = for {
-    config       <- ZIO.access[Config](_.get)
-    _            <- Logging.info(s"Loaded configuration: $config")
-    wsKey         = config.security.websocketKey.getOrElse(UUID.randomUUID().toString)
-    _            <- Logging.warn(securityWarning)
-    _            <- Logging.info(banner)
-    _            <- serve(wsKey).orDie
-  } yield 0
 
+  def main: ZIO[AppEnv, String, Int] = {
```

Comment:
The change here is that the args and global env are provided outside, in the `Main` entry point. `Server` is no longer a `polynote.app.App`. Probably `Server` could even be an `object` now, but I left it as a class anyway.